### PR TITLE
remove appexchange

### PR DESCRIPTION
**Pull request by illonage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,22 +52,6 @@ How you install the Tableau Viz Lightning web component determines how you need
 
 ---
 
-**Note:** If you installed the Tableau Viz Lightning component from AppExchange, you need to change the namespace the sample uses before you can deploy the sample to your org. Every component is part of a namespace. The sample project uses the default namespace `c`. In contrast, the Tableau Viz Lightning web component from AppExchange uses the namespace `tab`. Without modification, the sample will not work if you have installed the Tableau Viz Lightning web component from the AppExchange. You can skip this step if you didn't install the Tableau Viz web component from the AppExchange.
```
Comment (d45) [inline README.md:55]: @illonage  - I didn't see this earlier.  
So is the namespace mismatch no longer an issue with the Tableau Viz LWC from AppExchange (`tab` vs. `c`)?

Comment (illonage) [reply README.md:55]: Actually, it is not going to work if the tableau viz component is installed from AppExchange

Comment (d45) [reply README.md:55]: Oh. OK. We need to drop that option from the text.

Comment (illonage) [reply README.md:55]: Yes, i think i deleted this option from the text in this PR

**Comment by illonage** [comment]
@d45 ?

**Pull request merged**